### PR TITLE
Sorting ideas

### PR DESCRIPTION
**Pull request by vivienzhou2014**
Changes are in line 78-86, I changed the index of comment to get most recent comment created time and changed the data used for recentCommentTime to meet new requirement. Both changes are for recent activity sorting issue.

### CODE REVIEW

File: apps/ideaspace/src/components/modules/BrowseIdeas/BrowseIdeas.js [hunk 1]
```diff
@@ -75,10 +75,17 @@ function BrowseIdeas() {
       if (item?.comments?.data) {
         item.comments = cleanDataList(item.comments.data);
 
+        //add current time to created time to ensure ideas without comment will be listed first when sorted by recent activity
+        const totalMillis =
+          new Date().getTime() + new Date(item.createdAt).getTime();
+        const formatedCombinedTime = new Date(totalMillis).toISOString();
+        console.log('formated:' + formatedCombinedTime);
```
Comment (Ahsaask) [inline apps/ideaspace/src/components/modules/BrowseIdeas/BrowseIdeas.js]: Let's remove console.log if it's not necessary. Thank you!

**Review by Ahsaask — APPROVED** [review]
Looks good!

**Comment by Ahsaask** [comment]
I am not sure if we want to merge it into master. According to my understanding it should first go to development/ideaspace branch.

**Comment by dbradham** [comment]
I rebased this PR from `master` to `development/ideaspace` since we are in a code freeze. @vivienzhou2014 @chadcrotchett @Ahsaask

**Pull request merged**